internal/datanode/compactor_test.go

```diff
@@ -853,6 +855,9 @@ func (mfm *mockFlushManager) flushDelData(data *DelDataBuf, segmentID UniqueID,
 	if mfm.returnError {
 		return fmt.Errorf("mock error")
 	}
```

Comment:
add recordFlushedSeg to record all flushed segments in the process of running ut, aiming at verifying the related code is running correctly